File: DotNetGrc/Grc/Ast/Node/Func/VarIdentifierT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Grc.Ast.Visitor;

namespace Grc.Ast.Node.Func
{
	class VarIdentifierT : NodeBase
	{
		public VarIdentifierT(string text) : base(text)
		{
		}

		public override void Accept(IVisitor v)
		{
			v.Visit(this);
		}
	}
}

File: Parser/src/main/java/k31/grc/ast/node/func/VarIdentifierT.java
package k31.grc.ast.node.func;

import k31.grc.ast.node.NodeBase;
import k31.grc.ast.visitor.Visitor;

public class VarIdentifierT extends NodeBase {

	public VarIdentifierT(String text) {
		super(text);
	}

	@Override
	public void accept(Visitor v) {

		v.visit(this);
	}
}
